cmd/guacone/cmd/scorecard.go

```diff
@@ -204,4 +203,6 @@ func validateScorecardFlags(
 
 func init() {
 	certifierCmd.AddCommand(scorecardCmd)
+	scorecardCmd.Flags().Bool("use-scorecard-api", false, "use the scorecard API")
+	viper.BindPFlag("use-scorecard-api", scorecardCmd.Flags().Lookup("use-scorecard-api"))
```

Comment:
Could you add the flags in a similar manner to: https://github.com/guacsec/guac/blob/8b702d02d6fc63b263f19de5ef1cc835e722ac78/cmd/guacone/cmd/vulnerability.go#L628-L637

Reply:
Doing this should also fix the lint.

Reply:
Thanks for the review. I have fixed the problem!